### PR TITLE
Add "LumBEARjack" - fix issue #30

### PR DESCRIPTION
**Pull request by Pengu12345**
Added Fresh SFX's of the minigame "Blue Bear" 

Also , we need to make a better Icon for LumBEARJack .

### CODE REVIEW

**Comment by Pengu12345** [comment]
The minigame database seem to have errors , but I don't know why...  If someone would check or fix the error , it could be nice ^^

File: core/assets/data/games.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 [
+  "LumBearJack"
```
Comment (chrislo27) [inline core/assets/data/games.json]: Use `camelCase`. `lumBearJack`

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 2]
```diff
@@ -0,0 +1,149 @@
+{
+  "gameID": "lumBearJack",
+  "gameName": "LumBEARjack",
+  "series": "megamix",
+   "cues": [
+   {  
+   "id" : "lumBearJack/Nyah"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Syntax errors everywhere. Run this json file through a validator and fix all of them.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+	"cues": [
+	 {
+        "id": "lumBearJack/Nyah",
+        "beat": 0
+       },
+	   
+	   {
+	    "id": "lumBearJack/BigPose",
+        "beat": 1
+       },
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Formatting is completely off.

**Review by chrislo27 — CHANGES_REQUESTED** [review]
Please change the title of this pull request to accurately describe what changes you made. Overall, a very sloppy pull request that does not follow the file format naming style + the json has syntax errors everywhere.

File: core/assets/sounds/cues/LumBearJack/data.json (hunk 2 above)
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Again, IDs use `camelCase`. This should just be `nyah`.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 4]
```diff
@@ -0,0 +1,149 @@
+{
+  "gameID": "lumBearJack",
+  "gameName": "LumBEARjack",
+  "series": "megamix",
+   "cues": [
+   {  
+   "id" : "lumBearJack/Nyah"
+   "duration": 1
+   "name": "Nyah"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Names are also generally lower case.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+   {  
+   "id" : "lumBearJack/Nyah"
+   "duration": 1
+   "name": "Nyah"
+   },
+   
+      {  
+   "id" : "lumBearJack/Pose"
+   "duration": 1
+   "name": "Pose"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Lower case names.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/SingleHit"
+   "duration": 1
+   "name": "Single hit"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Lower case names.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/Double1"
+   "duration": 1
+   "name": "Double 1"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Lower case names.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/Double2"
+   "duration": 1
+   "name": "Double 2"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Lower case names.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 9]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/BigPose"
+   "duration": 1
+   "name": "Big Pose"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Lower case names.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/Big1"
+   "duration": 1
+   "name": "Big 1"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Lower case names. Use `big log 1`.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/Big2"
+   "duration": 1
+   "name": "Big 2"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Lower case names. Use `big log 2`.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 12]
```diff
@@ -0,0 +1,149 @@
...
+   "id" : "lumBearJack/Big3"
+   "duration": 1
+   "name": "Big 3"
+   },
+      {  
+   "id" : "lumBearJack/BigHitted"
+   "duration": 1.5
+   "name": "Big Hit"
+   }
+	],
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Formatting.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 13]
```diff
@@ -0,0 +1,149 @@
...
+      {  
+   "id" : "lumBearJack/Big2"
+   "duration": 1
+   "name": "Big 2"
+   },
+   
+      {  
+   "id" : "lumBearJack/Big3"
+   "duration": 1
+   "name": "Big 3"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Lower case names.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 14]
```diff
@@ -0,0 +1,149 @@
...
+   },
+      {  
+   "id" : "lumBearJack/BigHitted"
+   "duration": 1.5
+   "name": "Big Hit"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Lower case names.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 15]
```diff
@@ -0,0 +1,149 @@
...
+   }
+	],
+	
+	
+	
+	"patterns" : [
+		{
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Formatting.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 16]
```diff
@@ -0,0 +1,149 @@
...
+	   {
+	    "id": "lumBearJack/SingleHit",
+        "beat": 2
+       }
+	]
+		}
+		
+		{
+	"id": "lumBearJack_Double"
+	"name": "Double"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Lower case names.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 17]
```diff
@@ -0,0 +1,149 @@
...
+        "beat": 4
+       }
+	   	   {
+	    "id": "lumBearJack/BigHitted",
+        "beat": 5
+       }
+	 ]
+   }
+   ]
+  }
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Newline at end of file.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 18]
```diff
@@ -0,0 +1,149 @@
...
+	   
+	   {
+	    "id": "lumBearJack/Double2"
+	    "beat": 3
+	   }
+	]
+		}
+		
+			{
+	"id": "lumBearJack_Big"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Use `camelCase` for IDs.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 19]
```diff
@@ -0,0 +1,149 @@
...
+       }
+	]
+		}
+		
+		{
+	"id": "lumBearJack_Double"
+	"name": "Double"
+	"cues": [
+	 {
+        "id": "lumBearJack/Nyah",
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Use `camelCase` for IDs.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 20]
```diff
@@ -0,0 +1,149 @@
...
+	
+	
+	
+	"patterns" : [
+		{
+	"id": "lumBearJack_Single"
+	"name": "single"
+	"cues": [
+	 {
+        "id": "lumBearJack/Nyah",
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Use `camelCase` for IDs.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 21]
```diff
@@ -0,0 +1,149 @@
...
+        "beat": 0
+       },
+	   
+	   {
+	    "id": "lumBearJack/Pose",
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Use `camelCase` for IDs.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 22]
```diff
@@ -0,0 +1,149 @@
...
+        "beat": 1
+       },
+	   
+	   {
+	    "id": "lumBearJack/SingleHit",
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Use `camelCase` for IDs.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 23]
```diff
@@ -0,0 +1,149 @@
...
+        "beat": 2
+       }
+	]
+		}
+		
+		{
+	"id": "lumBearJack_Double"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Use `camelCase` for IDs.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 24]
```diff
@@ -0,0 +1,149 @@
...
+	"name": "Double"
+	"cues": [
+	 {
+        "id": "lumBearJack/Nyah",
+        "beat": 0
+       },
+	   
+	   {
+	    "id": "lumBearJack/Pose",
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Use `camelCase` for IDs.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 25]
```diff
@@ -0,0 +1,149 @@
...
+        "beat": 1
+       },
+	   
+	   {
+	    "id": "lumBearJack/Double1",
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Use `camelCase` for IDs.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 26]
```diff
@@ -0,0 +1,149 @@
...
+        "beat": 2
+       }
+	   
+	   {
+	    "id": "lumBearJack/Double2"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: Use `camelCase` for IDs.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 27]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/BigPose"
+   "duration": 1
+   "name": "Big Pose"
+   },
+   
+      {  
+   "id" : "lumBearJack/Big1"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: IDs use `camelCase`.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 28]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/Double2"
+   "duration": 1
+   "name": "Double 2"
+   },
+   
+      {  
+   "id" : "lumBearJack/BigPose"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: IDs use `camelCase`.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 29]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/Double1"
+   "duration": 1
+   "name": "Double 1"
+   },
+   
+      {  
+   "id" : "lumBearJack/Double2"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: IDs use `camelCase`.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 30]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/SingleHit"
+   "duration": 1
+   "name": "Single hit"
+   },
+   
+      {  
+   "id" : "lumBearJack/Double1"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: IDs use `camelCase`.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 31]
```diff
@@ -0,0 +1,149 @@
...
+   },
+   
+      {  
+   "id" : "lumBearJack/Pose"
+   "duration": 1
+   "name": "Pose"
+   },
+   
+      {  
+   "id" : "lumBearJack/SingleHit"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: IDs use `camelCase`.

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 32]
```diff
@@ -0,0 +1,149 @@
...
+  "series": "megamix",
+   "cues": [
+   {  
+   "id" : "lumBearJack/Nyah"
+   "duration": 1
+   "name": "Nyah"
+   },
+   
+      {  
+   "id" : "lumBearJack/Pose"
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json]: IDs use `camelCase`.

**Review by chrislo27 — CHANGES_REQUESTED** [review]
More JSON style errors

**Comment by Pengu12345** [comment]
I see , sorry :c 

I will try to fix theses problems ^^'

**Comment by Pengu12345** [comment]
I tried to fix the problems.
If there are any more problems , please tell me ^^

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 33]
```diff
@@ -0,0 +1,129 @@
...
+   },   
+   {  
+     "id" : "lumBearJack/pose",
+     "duration": 1,
+     "name": "pose",
+   },   
+   {  
+     "id" : "lumBearJack/singleHit",
+     "duration": 1,
+     "name": "pingle hit",
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json:19]: pingle

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 34]
```diff
@@ -0,0 +1,129 @@
...
+     "duration": 1,
+     "name": "big log 3",
+   },	   
+   {  
+     "id" : "lumBearJack/bigHitted",
+     "duration": 1.5,
+     "name": "big hit",
+   }
+],	
+"patterns" : [
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json:57]: Formatting

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 35]
```diff
@@ -0,0 +1,129 @@
...
+            {
+	     "id": "lumBearJack/pose",
+             "beat": 1
+            },	   
+            {
+	    "id": "lumBearJack/singleHit",
+            "beat": 2
+            }
+               ]
+},
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json:75]: Formatting

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 36]
```diff
@@ -0,0 +1,129 @@
...
+	     "id": "lumBearJack/double1",
+             "beat": 2
+            },	   
+	    {
+	     "id": "lumBearJack/double2",
+	     "beat": 3
+	    }
+	        ]
+},		
+{
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json:98]: Formatting

File: core/assets/sounds/cues/LumBearJack/data.json [hunk 37]
```diff
@@ -0,0 +1,129 @@
...
+            },
+	    {
+	     "id": "lumBearJack/big3",
+             "beat": 4
+            },
+	    {
+	     "id": "lumBearJack/bigHitted",
+             "beat": 5
+            }
+	         ]
```
Comment (chrislo27) [inline core/assets/sounds/cues/LumBearJack/data.json:126]: Formatting

**Review by chrislo27 — CHANGES_REQUESTED** [review]
You didn't change the camelCase requirements on the folder name nor the sound file names. There's still formatting errors.

**Comment by chrislo27** [comment]
#46 covers this.

**Pull request closed without merging**